docs/templates/table.md.go.tpl

```diff
@@ -23,6 +23,10 @@ It supports incremental syncs
 {{- end -}}.
 {{- end -}}
 
+{{- if $.IsPaid -}}
+This is a premium table. To sync this table you must be logged in via `cloudquery login` or you must use a valid API Key which can be generated at `cloud.cloudquery.io`
```

Comment:
Are we sure about the backticks around `cloud`? Why not make it a link (by either leaving it without backticks, or possibly by markdown syntax)